docker/celestia/run-validator.sh

```diff
@@ -147,12 +147,12 @@ setup_private_validator() {
   dasel put -f "$CONFIG_DIR/config/config.toml" -t string -v '0.0.0.0:26660' instrumentation.prometheus_listen_addr
 
   # Adjusting for faster block times. Default values are in comments above
-  # Currently disabled
+  # Currently disabled for being closer to actual mocha
```

Comment:
```suggestion
  # Currently disabled to be closer to actual mocha
```
maybe? (very minor nit, non-blocking if you'd rather merge faster)

Reply:
No rush, good grammar is important